Bitly/Bitly_Get_Metrics_for_a_Bitlink_by_Country.ipynb

```diff
@@ -0,0 +1,241 @@
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad2305e2-61b2-471b-9ffb-f18feb9fbfbe",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "import requests\nimport json",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "921c9171-fd42-4479-a247-60ead18c5f48",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup Variables\n- **access_token**: [Generate an access token](https://dev.bitly.com/authentication.html)\n- **bitlink**: The Bitlink for which you want to get metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72bc86f7-f3fa-41c1-93d3-d1815d82f518",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "access_token = \"YOUR_ACCESS_TOKEN\"\nbitlink = \"YOUR_BITLINK\"",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74c7e81f-f3ec-4adb-925a-884a1d6ff0e1",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f78ddf67-4e3f-4c2c-a0b9-b3506f57a295",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Get Metrics for a Bitlink by Country"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "753fb196-3a3f-487b-8f90-751a85740734",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "This function will use the Bitly API to get metrics for a Bitlink by Country."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d77bd63a-fcce-4c96-9c60-111c70796b0d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "def get_metrics_by_country(access_token, bitlink):\n    url = f\"https://api-ssl.bitly.com/v4/bitlinks/{bitlink}/countries\"\n    headers = {\"Authorization\": f\"Bearer {access_token}\"}\n    response = requests.get(url, headers=headers)\n    if response.status_code == 200:\n        return json.loads(response.content)\n    else:\n        return None",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a473593-980a-4f42-a3b3-a2380c22e681",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4498312-7691-445a-a4cb-84afd4c7b15b",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Display result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "22effd68-8e9a-4890-a967-c8801ac8a2ba",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "metrics_by_country = get_metrics_by_country(access_token, bitlink)\nif metrics_by_country:\n    print(json.dumps(metrics_by_country, indent=4))\nelse:\n    print(\"An error occurred.\")",
```

Comment:
<img width="1337" alt="Screenshot 2023-02-24 at 7 14 52 PM" src="https://user-images.githubusercontent.com/81156510/221193854-d8a31335-6683-41af-8d7f-70cc2d3fd2cd.png">

No explicit errors as it is been caught using `if-else` but no metrics were generated.